app/src/main/java/com/framgia/forder/screen/shopmanagement/ShopManagementFragment.java

```diff
@@ -17,6 +18,8 @@
 import com.framgia.forder.data.source.remote.api.service.FOrderServiceClient;
 import com.framgia.forder.databinding.FragmentShopmanagementBinding;
 import com.framgia.forder.utils.navigator.Navigator;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports